Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3009,6 +3009,56 @@ internal static bool ShouldDrawWarningIconForBinding(string bindingPath)
 
         #region Actions
 
+#if UNITY_INPUT_SYSTEM_PROJECT_WIDE_ACTIONS
+        /// <summary>
+        /// The set of project-wide available input actions.
+        /// </summary>
+        private static InputActionAsset s_projectWideActions;
+        public static InputActionAsset actions
+        {
+            get
+            {
+                if (s_projectWideActions != null)
+                    return s_projectWideActions;
+
+                #if UNITY_EDITOR
+                // Load the InputActionsAsset and store it in EditorBuildSettings so it can be packed in Player builds
+                s_projectWideActions = Editor.ProjectWideActionsAsset.GetOrCreate();
+                if (!string.IsNullOrEmpty(AssetDatabase.GetAssetPath(s_projectWideActions)))
+                {
+                    EditorBuildSettings.AddConfigObject(InputSettingsProvider.kEditorBuildSettingsActionsConfigKey,
+                        s_projectWideActions, true);
+                }
+                #else
+                s_projectWideActions = Resources.FindObjectsOfTypeAll<InputActionAsset>().FirstOrDefault();
```

Comment:
Is this line valid - could it end up picking up some other input action asset that we don't expect ?

Reply:
Can the resources areas only contain the pre built one or could a user tuck some in there too ?
Would it be better to iterate over all and match by name.